src/pages/Home/Content/Content.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { ContentStyle } from "./ContentStyle";
+import { FoodItem, Category } from "DummyData/DataTypes";
+import { McSpicy } from "DummyData/McSpicy";
+import { OvaltineMcFlurry } from "DummyData/OvaltineMcFlurry";
+import { Categories } from "DummyData/Categories";
+import SectionHeader from "./Section/SectionHeader/SectionHeader";
+import ItemList from "./Section/ItemList/ItemList";
+
+const AllFoodItems: FoodItem[] = [
+  McSpicy,
+  McSpicy,
+  McSpicy,
+  McSpicy,
+  McSpicy,
+  McSpicy,
+  OvaltineMcFlurry,
+  OvaltineMcFlurry,
+  OvaltineMcFlurry,
+];
+
+//trying to render 6 cards of identical McSpicy foodItems. Should be under the "recommended" SectionHeader.
+//McSpicy would appear both under "Recommended" and "Main Course". FoodItem can have more than one category.
+export default function Content() {
+  return (
+    <ContentStyle>
+      {Categories.map((sectionHeader) => {
```

Comment:
logic like that should not be in the return of the component. It should be handled above to create an array of JSX components which are mapped out in the return function of Content

Reply:
Fixed this along with the key problem.